feedback_sentry/lib/feedback_sentry.dart

```diff
@@ -26,10 +26,6 @@ extension SentryFeedbackX on FeedbackController {
   /// This method opens the feedback ui and the users feedback
   /// is uploaded to Sentry after the user submitted his feedback.
   /// [name] and [email] are optional. They are shown in the Sentry.io ui.s
-  @Deprecated(
```

Comment:
What's the reason for removing this deprecation notice?

Reply:
if understood the deprecation reason correctly ofc

Reply:
I don't even remember myself, but your explanation makes sense, so I'm good with the change 👍